packages/react-components/react-storybook-addon/src/decorators/withAriaLive.tsx

```diff
@@ -1,17 +1,34 @@
+import { AriaLiveAnnouncer } from '@fluentui/react-aria';
 import * as React from 'react';
 
-import { AriaLiveAnnouncer } from '@fluentui/react-aria';
+import { FluentStoryContext } from '../hooks';
+
+export const withAriaLive = (Story: () => JSX.Element, context: FluentStoryContext) => {
+  const { parameters } = context;
+  const { mode } = parameters;
+  const isVrTest = mode === 'vr-test';
 
-export const withAriaLive = (StoryFn: () => JSX.Element) => {
-  return <AriaLiveWrapper>{StoryFn()}</AriaLiveWrapper>;
+  // - VR tests don't need the AriaLiveAnnouncer
+  // - VR tests tooling supports only functions as story renderers :(
+  if (isVrTest) {
```

Comment:
i'm wondering if having ability to actually configure on addon registration level which decorators should be loaded would be a more robust solution then having this internal "hack" per say.

if we register decorators via storybook function name of decorator would be exposed which could be then used.

https://storybook.js.org/docs/6/addons/addons-api#makedecorator-api

WDYT ?

Reply:
Not sure that I follow, @Hotell can you please expand with a code example?

Reply:
sure. (NOTE: that makeDecoratorApi is not needed - I checked )

**Option 1:**

<img width="1704" alt="image" src="https://github.com/user-attachments/assets/3263c8f2-43d3-45b9-a43c-1682d718283a">


**Options 2:**

similar to yours but explicit public API  from addon POV

<img width="1690" alt="image" src="https://github.com/user-attachments/assets/e3aa859e-d9af-4a38-b88c-6ec75acb12e5">

Reply:
@Hotell I implemented option 2 👍